changelogs/fragments/1830-valmod_docmissingtype_batch1.yml

```
@@ -0,0 +1,11 @@
+bugfixes:
+  - kibana_plugin - ``state`` parameter choices must use ``list()`` in python3
```

Generally, all fragments should look like this:
```suggestion
  - kibana_plugin - ``state`` parameter choices must use ``list()`` in python3 (https://github.com/ansible-collections/community.general/pull/1830).
```